prisma/schema.prisma

```diff
@@ -25,14 +25,14 @@ model EventType {
   hidden                  Boolean                @default(false)
   users                   User[]                 @relation("user_eventtype")
   userId                  Int?
-  team                    Team?                  @relation(fields: [teamId], references: [id])
+  team                    Team?                  @relation(fields: [teamId], references: [id], onDelete: Cascade, onUpdate: Cascade)
```

Comment:
Deletes `EventType` if the correlated team is deleted

Reply:
makes sense